src/Mailgun/Messages/BatchMessage.php

```diff
@@ -103,6 +103,20 @@ protected function addRecipient($headerName, $address, $variables)
         }
         $this->batchRecipientAttributes["$address"] = $variables;
     }
+    
+     /**
+     * @param string     $address
+     * @param array|null $variables
+     * @return mixed
+     */
+    public function addToRecipient($address, $variables = null)
```

Comment:
If `$variables` should be an `array` then we should type hint it properly:
`array $variables = null`